tests/functional_test.js

```diff
@@ -215,15 +214,15 @@ describe('functional', function () {
         var publicURIs = [];
 
         walk.filesSync(path.join(__dirname, '..', 'public'), function (base, name) {
-            var root = base.split('/public/')[1];
+            var root = base.split(path.normalize('/public/'))[1];
 
             // ensure file is in whitelisted directory
             if (typeof root === 'undefined' || whitelist.indexOf(root.split(path.sep)[0]) === -1) {
                 return;
             }
 
             var domain = helpers.domainCheck('https://maxcdn.bootstrapcdn.com/');
-            var uri = domain + root + '/' + name;
+            var uri = helpers.normalizeURL(domain + root + '/' + name);
```

Comment:
Briefly played with both and it doesn't look like it, at least not with an equal amount of massaging.